Integrations/integration-QRadar.yml

```diff
@@ -64,7 +64,7 @@ script:
     var server = params.server+'/';
     var insecure = params.insecure;
     var query = params.query;
-    var offensesPerCall = parseInt(params.offensesPerCall);
+    var offensesPerCall = parseInt(params.offensesPerCall || 50);
```

Comment:
approved, but why is this needed? there's a default value for this parameter

Reply:
For already configured instances. Must always think about upgrade.

Reply:
right, that's why the default is there
if there's no configured value, the default should be used - why is it not being used after an upgrade?

Reply:
Because that's the way it works at the moment

Reply:
https://demisto.slack.com/archives/C1X1QDL6B/p1527163184000039

Reply:
so let's fix it :)